panel/util.py

```diff
@@ -9,9 +9,13 @@
 import numbers
 import datetime as dt
 
-from collections import defaultdict, MutableSequence, MutableMapping, OrderedDict
 from datetime import datetime
 from six import string_types
+from collections import defaultdict, OrderedDict
+try:
+    from collections.abc import MutableSequence, MutableMapping
+except ImportError:  # support for python>3.8
+    from collections import MutableSequence, MutableMapping
```

Comment:
Isn't this comment backwards?  I.e., isn't the `try` clause (collections.abc) required for python>3.8, and the `except` clause is for some older version (<3.3)?

Reply:
yeah, I put the comparison the wrong way round, should have been `python<3.8`

Reply:
oh, it was already merged. I'll include it in the docs PR

Reply:
Ok, but it's not true the other way around either; the `except` isn't needed for python<3.8, only for python<3.3.

Reply:
Ah, thanks for pointing that out.